### PR TITLE
Xdr-lite-integration-docs

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9678

## Description
Updated the description of the 'Cortex XDR - Lite Incident Handling' playbook.
Updated the Integration Documentation.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Integrations/CortexXDRIR/README.md [hunk 1]
```diff
@@ -65,6 +65,13 @@ and an analyst is assigned for manual investigation. If chosen, automated remedi
...
+
+To utilize this playbook for handling XDR incidents, the classifier should be empty, and the selected incident type should be `Cortex XDR - Lite`.
```
Comment (altmannyarden) [inline Packs/CortexXDR/Integrations/CortexXDRIR/README.md:74]: what about the mapper?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ description: |
 
   Verdict Handling:
   - Handles malicious incidents by initiating appropriate response actions, including blocking malicious indicators, isolating endpoints, and disabling user accounts.
+  
+  To utilize this playbook as the default for handling XDR incidents, the classifier should be empty, and the selected incident type should be `Cortex XDR - Lite`.
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling.yml:20]: what about the mapper here as well

Comment (altmannyarden) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling.yml:20]: also add, In the integration configuration.....

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

File: Packs/CortexXDR/Integrations/CortexXDRIR/README.md (hunk 1 above)
Comment (OmriItzhak) [reply Packs/CortexXDR/Integrations/CortexXDRIR/README.md:74]: Done

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling.yml (hunk 2 above)
Comment (OmriItzhak) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling.yml:20]: done

**Pull request merged**